### DISCUSSION TITLE
Let user define precise search string by (a) phrase search (b) logical operator OR

### DISCUSSION THREAD

**Discussion by raindropsfromsky** [Flow-Launcher/Flow.Launcher#1193] [Ideas]
Let user define his search string more precisely by:

(a) phrase search: FL must search for a precise phrase instead of searching for specified words in any random order. 
(b) logical operator OR: Let user connect words/phrases with OR so that FL can search for at least one of them.

If OR is not used, it implies that the search result must have all the words/phrases provided in the search.
In other words, AND is a default operator, which the user need not enter explicitly.

**Comment by taooceros** [comment]
you want to search files or programs?
